app/views/teachers/teacher-dashboard/BaseResourceHub/BaseResourceHubMainComponent.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="base-resource-hub">
+    <raw-pug-component :pug="teacherDashboardNavTemplate"></raw-pug-component>
+    <base-resource-hub />
+  </div>
+</template>
+
+<script>
+import BaseResourceHub from './index'
```

Comment:
Can we move this (and the ButtonResourceIcon component) from /ozaria to a shared location, instead of copying, while refactoring any differences that need to exist between the two? Our goal is to converge the CodeCombat and Ozaria code over time, so I don't want to introduce new copies.